### PR TITLE
feat: override command args of karma container

### PR DESCRIPTION
**Pull request by thorker**
The last missing piece to override basicauth credential without storing the whole configuration into a secret.

### CODE REVIEW

**Comment by thorker** [comment]
Can you please review, @desaintmartin ?

File: charts/karma/values.yaml [hunk 1]
```diff
@@ -239,3 +239,6 @@ extraVolumes: []
 
 # extra volumeMounts for the deployment
 extraVolumeMounts: []
+
+# override command args of karma container
+configurations: []
```
Comment (desaintmartin) [inline charts/karma/values.yaml:244]: Hmmm I would name it... args?

File: charts/karma/templates/deployment.yaml [hunk 2]
```diff
@@ -64,6 +64,14 @@ spec:
           {{- end }}
           {{- with .Values.env }}
 {{ toYaml . | indent 10 }}
+          {{- end }}
+          {{- if gt (len .Values.configurations) 0 }}
+          command:
```
Comment (desaintmartin) [inline charts/karma/templates/deployment.yaml:69]: Is command mandatory when setting args? I have a doubt here!

**Review by desaintmartin — CHANGES_REQUESTED** [review]
Thanks for the contribution! Here are a few comments.

File: charts/karma/values.yaml (hunk 1 above)
Comment (thorker) [reply charts/karma/values.yaml]: done

File: charts/karma/templates/deployment.yaml (hunk 2 above)
Comment (thorker) [reply charts/karma/templates/deployment.yaml]: You are right. Tested it without the command and still fine.

**Review by desaintmartin — APPROVED** [review]
Thanks a lot!

**Pull request merged**